core/_interfaces/test-complete.i.ts

```diff
@@ -0,0 +1,14 @@
+import { ITestCase, ITest, ITestFixture } from "./index";
```

Comment:
Could we move this file to a new `core/events` folder and put in a little `index.ts` namespace please. As I think we'll be adding more events too and it'd be nice to keep them together.